tests/include/igl/randperm.cpp

```diff
@@ -0,0 +1,70 @@
+#include <test_common.h>
+#include <igl/randperm.h>
+#include <random>
+
+TEST(randperm, default_rng_reproduce_identity)
```

Comment:
Please take a look at [other tests](https://github.com/libigl/libigl/blob/dev/tests/include/igl/cotmatrix.cpp) in the `dev` branch, since we've switched to Catch2 for the unit tests (this is why it doesn't compile right now).

Reply:
I forgot to merge to code to dev branch in my machine... Thank you for letting me know!